packages/mui-utils/src/getDisplayName/getDisplayName.test.tsx

```diff
@@ -41,13 +41,13 @@ describe('utils/getDisplayName.js', () => {
       ));
       NamedForwardRefComponent.displayName = 'Div';
 
-      const AnonymousMemoComponent = React.memo((props, ref) => <div {...props} ref={ref} />);
+      const AnonymousMemoComponent = React.memo((props) => <div {...props} />);
```

Comment:
Was not caught previously because types assumed the `ref` argument is actually the legacy context. Now that we removed legacy context, this bug is surface.